charts/aws-ebs-csi-driver/templates/poddisruptionbudget-controller.yaml

```diff
@@ -11,9 +11,16 @@ spec:
     matchLabels:
       app: ebs-csi-controller
       {{- include "aws-ebs-csi-driver.selectorLabels" . | nindent 6 }}
+  {{- if (semverCompare ">= 1.27-0" .Capabilities.KubeVersion.Version) }}
```

Comment:
Because this is an optional parameter, drop this `if` statement, it's not needed